src/test/java/com/synopsys/integration/alert/channel/ChannelDescriptorTestIT.java

```diff
@@ -53,43 +63,54 @@
     protected DistributionEvent channelEvent;
 
     protected ContentConverter contentConverter;
-    protected TestProperties properties;
+    protected TestProperties testProperties;
 
+    @Autowired
+    protected ProviderKey providerKey;
+    @Autowired
+    protected JobAccessorV2 jobAccessor;
     @Autowired
     protected DefaultConfigurationAccessor configurationAccessor;
     @Autowired
     protected DefaultDescriptorAccessor descriptorAccessor;
     @Autowired
+    protected DescriptorProcessor descriptorProcessor;
+    @Autowired
     protected RegisteredDescriptorRepository registeredDescriptorRepository;
 
-    protected ConfigurationModel provider_global;
-    protected Optional<ConfigurationModel> global_config;
-    protected ConfigurationModel distribution_config;
-    protected String destinationName;
+    protected ConfigurationModel providerGlobalConfig;
+    protected Optional<ConfigurationModel> optionalChannelGlobalConfig;
+    protected DistributionJobModel distributionJobModel;
+    protected String eventDestinationName;
 
     @BeforeEach
     public void init() throws Exception {
         gson = new Gson();
         contentConverter = new ContentConverter(gson, new DefaultConversionService());
-        properties = new TestProperties();
-        global_config = saveGlobalConfiguration();
-        distribution_config = saveDistributionConfiguration();
+        testProperties = new TestProperties();
+        providerGlobalConfig = saveProviderGlobalConfig();
+        optionalChannelGlobalConfig = saveGlobalConfiguration();
+        eventDestinationName = getEventDestinationName();
         channelEvent = createChannelEvent();
-        destinationName = getDestinationName();
+        distributionJobModel = saveDistributionJob();
     }
 
     @AfterEach
     public void cleanupTest() {
-        if (null != global_config && global_config.isPresent()) {
-            configurationAccessor.deleteConfiguration(global_config.get());
+        if (null != optionalChannelGlobalConfig && optionalChannelGlobalConfig.isPresent()) {
```

Comment:
Why would the optional be null?

Reply:
I believe this code was here before and I just renamed the variable. I'll clean this up though.